src/legacy/core_plugins/kibana/public/management/sections/objects/lib/__jest__/resolve_saved_objects.test.ts

```diff
@@ -23,20 +23,29 @@ import {
   saveObjects,
   saveObject,
 } from '../resolve_saved_objects';
-
-jest.mock('../../../../../../../../../plugins/kibana_utils/public', () => ({
-  SavedObjectNotFound: class SavedObjectNotFound extends Error {
-    constructor(options) {
-      super();
-      for (const option in options) {
-        if (options.hasOwnProperty(option)) {
-          this[option] = options[option];
-        }
```

Comment:
This mocking was useless, as the only runtime usage of the mock was the import in the test file... adapted to declare the `SavedObjectNotFound` in the test file instead